packages/@glimmer/core/src/managers/component/template-only.ts

```diff
@@ -1,10 +1,8 @@
 import {
```

Comment:
Just curious, this whole file / concept should be upstreamed right?

Reply:
Yes, it will be after a few more refactors